exercises/pythagorean-triplet/example.jl

```diff
@@ -5,7 +5,7 @@ function pythagorean_triplets(input)
             c = input - a - b
             triplet = sort([a, b, c])
             if (triplet[1]^2 + triplet[2]^2 == triplet[3]^2)
-                push!(triplets, Int32.(triplet))
+                push!(triplets, tuple(Int.(triplet)))
```

Comment:
This creates a tuple containing 1 array:

```julia
julia> tuple(Int.(triplet))
([1, 2, 3],)
```

You want to use `Tuple(Int.(triplet))` or `tuple(Int.(triplet)...)` here.

Reply:
Thanks